Finish AJAX request/app.js

```diff
@@ -0,0 +1,10 @@
+const request = new XMLHttpRequest();
+request.onreadystatechange = function () {
+  if (request.readyState === 4) {
+    document.getElementById("footer").innerHTML = request.responseText;
+  }
+};
+
+request.open('GET', 'footer.html');
```

Comment:
I think this should be

```suggestion
request.open('GET', 'index.html');
```

Since `footer.html` doesn't exist on this folder